### ISSUE TITLE
[Feature request]REX LR scheduler

### ISSUE THREAD

**Issue by sdbds** [kozistr/pytorch_optimizer#217]
## Paper or Code

REX LR scheduler
From https://arxiv.org/abs/2107.04197

Implementation is based on https://github.com/Nerogar/OneTrainer/blob/2c6f34ea0838e5a86774a1cf75093d7e97c70f03/modules/util/lr_scheduler_util.py#L66

**Comment by kozistr** [comment]
thanks! I'll look into it :)

**Comment by kozistr** [comment]
now you can check the implementation on the latest `main` branch :)
